src/reporters/base-reporter.js

```diff
@@ -210,7 +220,7 @@ export default class BaseReporter {
   header(command: string, pkg: Package) {}
 
   // the screen shown at the very end of the CLI
-  footer(showPeakMemory: boolean) {}
+  footer(showPeakMemory: boolean, showNetworkUsage: boolean) {}
```

Comment:
Oh, now I see what is happening above. Please avoid `boolean` arguments and convert this into an `options` object:
```
footer({showPeakMemory?: boolean, showNetworkUsage?: boolean} = {showPeakMemory: false, showNetworkUsage: false})
```